### ISSUE TITLE
UnicodeDecodeError when a png file is included in the template

### ISSUE THREAD

**Issue by goldginkgo** [Roche/foxops#384]
Our template repository requires a png file in it, but it still raises an error when the png file is defined in excluded_files section.
```
Traceback (most recent call last):
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/__main__.py", line 120, in cmd_initialize
    asyncio.run(
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/initialization.py", line 38, in initialize_incarnation
    return await _initialize_incarnation(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/initialization.py", line 88, in _initialize_incarnation
    await render_template(
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/rendering.py", line 106, in render_template
    await _render_template_file(
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/rendering.py", line 85, in _render_template_file
    return await render_template_file(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/site-packages/foxops/engine/rendering.py", line 132, in render_template_file
    rendered_content = template_file_path.read_text()
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daiy24/.pyenv/versions/3.11.4/lib/python3.11/pathlib.py", line 1059, in read_text
    return f.read()
           ^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x89 in position 0: invalid start byte
```

**Comment by goldginkgo** [comment]
Thanks for merging the code! @defreng 
Currently I'm using fengine as a library, can you publish a new package so that the latest fix is included?

**Comment by defreng** [comment]
Hi @goldginkgo 

I just released 2.0.0-alpha5 ... Make sure that you use the pip install `--pre` option when installing
